frontend/src/components/AdminRequests.js

```diff
@@ -1,220 +1,90 @@
-                      DATE CHANGE
-                      <span style={textBreakStyle}></span>
-                      5/7 - 7/7 -&gt; 7/7 - 9/7
-                    </Typography>
-                  </Box>
-                </Paper>
-              </Grid>
-              <Grid item xs={12}>
-                <Paper elevation={0} style={gridItemStyle.gridItem2}>
-                  <Box paddingLeft="10px">
-                    <Typography
-                      variant="h5"
-                      align="left"
-                      sx={{ fontWeight: "bold" }}
-                    >
-                      USER 1
-                    </Typography>
-                    <Divider width="200px" />
-                    <Typography
-                      variant="h6"
-                      align="left"
-                      sx={{ fontWeight: "bold" }}
-                    >
-                      DATE CHANGE
-                      <span style={textBreakStyle}></span>
-                      5/7 - 7/7 -&gt; 7/7 - 9/7
-                    </Typography>
-                  </Box>
-                </Paper>
-              </Grid>
-              <Grid item xs={12}>
-                <Paper elevation={0} style={gridItemStyle.gridItem3}>
-                  <Box paddingLeft="10px">
-                    <Typography
-                      variant="h5"
-                      align="left"
-                      sx={{ fontWeight: "bold" }}
-                    >
-                      USER 1
-                    </Typography>
-                    <Divider width="200px" />
-                    <Typography
-                      variant="h6"
-                      align="left"
-                      sx={{ fontWeight: "bold" }}
-                    >
-                      DATE CHANGE
-                      <span style={textBreakStyle}></span>
-                      5/7 - 7/7 -&gt; 7/7 - 9/7
-                    </Typography>
-                  </Box>
-                </Paper>
-              </Grid>
-              <Grid item xs={12}>
-                <Paper elevation={0} style={gridItemStyle.gridItem2}>
-                  <Box paddingLeft="10px">
-                    <Typography
-                      variant="h5"
-                      align="left"
-                      sx={{ fontWeight: "bold" }}
-                    >
-                      USER 1
-                    </Typography>
-                    <Divider width="200px" />
-                    <Typography
-                      variant="h6"
-                      align="left"
-                      sx={{ fontWeight: "bold" }}
-                    >
-                      DATE CHANGE
-                      <span style={textBreakStyle}></span>
-                      5/7 - 7/7 -&gt; 7/7 - 9/7
-                    </Typography>
-                  </Box>
-                </Paper>
-              </Grid>
+              {requests.map((request, index) => (
+                <RequestBlock
+                  key={request.user_id}
```

Comment:
A user ID isn't a unique key (a user can have multiple requests). We should use the request document ID returned by Firebase, something like
```js
key={request._id}
```

Reply:
is it okay to just use the booking id? thats what ive set it as right now